spec/virtual_total_spec.rb

```diff
@@ -328,4 +361,65 @@ def model_with_children(count)
       end
     end
   end
+
+  describe ".virtual_aggregation" do
```

Comment:
Upon further inspection, not really, since they are both part of the same file.

Just seems like `virtual_total` inherits from `virtual_aggregation` conceptually, but instead, it is the reverse.

For now, carry on.

Reply:
:( `virtual_total` calls `virtual_aggregate` - so yea, total should inherit from aggregation.

ideas how to make this more clear (probably a separate PR)